src/algebra/module/linear_map.lean

```diff
@@ -326,12 +326,19 @@ def add_monoid_hom.to_int_linear_map [add_comm_group M] [add_comm_group M₂] (f
   M →ₗ[ℤ] M₂ :=
 ⟨f, f.map_add, f.map_int_module_smul⟩
 
+lemma add_monoid_hom.coe_to_int_linear_map [add_comm_group M] [add_comm_group M₂] (f : M →+ M₂) :
```

Comment:
```suggestion
@[simp] lemma add_monoid_hom.coe_to_int_linear_map [add_comm_group M] [add_comm_group M₂] (f : M →+ M₂) :
```

Reply:
I wasn't sure about this one; `linear_map.coe_to_add_monoid_hom` is always a good simp lemma, because after applying it there are more simp lemmas that apply not fewer. If I make this a simp lemma though, then `F.to_int_linear_map (r \smul x)` won't simplify to `r \smul F.to_int_linear_map x` as `simp` will first throw away the information that the map is linear.

Reply:
I see. On the other hand, in 9/10 cases this will leave an ugly looking goal, and the user has to hunt down this lemma manually.
If we add it to the simp set, then in the occasional place where it is "bad", a user can quickly discover this using `squeeze_simp`, and then locally remove if from the simp set using a `squeeze_simp [- this_lemma]`.

Reply:
I see your point - indeed, `simp` is often my first choice for finding the lemma I need, as its way faster than library search and gets rid of stuff I don't care about. Shall I go ahead and make the change? The place where I needed this lemma was a `rw` anyway, so this won't affect my other PRs.

Reply:
I would vote for adding `@[simp]`.

Reply:
Done, mine was only a very weak vote against anyway.